frontend/src/context/ThemeContext.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useState, useEffect } from "react";
+
+export const ThemeContext = createContext();
+
+const ThemeContextProvider = ({ children }) => {
+  const [theme, setTheme] = useState(
```

Comment:
Putting this in context is really nice. But `theme` is basically a boolean variable so maybe something like `[isDarkMode, setDarkMode]` will be more appropriate?

Reply:
done